wp-content/civi-extensions/goonjcustom/Civi/CollectionCampService.php

```
@@ -1853,7 +1853,24 @@ public static function sendCampOutcomeAckEmailAfter5Days($collectionCamp) {
+    if (!empty($collectionSourceVehicleDispatche)) {
+        // Outer bullet
+        $materialGeneratedHtml .= "<li>Material generated:<br>";
+        // Inner numbered list with inline style for emails
+        $materialGeneratedHtml .= "<ol style='margin:0; padding-left:20px;'>";
+        foreach ($collectionSourceVehicleDispatche as $dispatch) {
+            $materialGeneratedHtml .= "<li>" . $dispatch['Acknowledgement_For_Logistics.No_of_bags_received_at_PU_Office'] . "</li>";
+        }
+        $materialGeneratedHtml .= "</ol></li>";
+    }
 
     $materialGenerated = $collectionSourceVehicleDispatche['Acknowledgement_For_Logistics.No_of_bags_received_at_PU_Office'];
```

PHPMD (2.15.0)

Avoid unused local variables such as '$materialGenerated'. (undefined)

(UnusedLocalVariable)